### PR TITLE
Exercises

### PR DESCRIPTION
**Pull request by hilahersz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
...
+[![run unittests](https://github.com/advanced-python-ds/code-as-data/actions/workflows/run_unittests.yml/badge.svg)](https://github.com/advanced-python-ds/code-as-data/actions/workflows/run_unittests.yml)
+
+
+## This is Advanced Python for Data Science
+### Intro Course: Code as Data
+
+In this repo, you can find the coding exercises for the course.
+You may use this template repo to work on your code locally. 
+
+Link to the course:
```
Comment (alhurwitz) [inline README.md:14]: Do we have the link to the course?

File: README.md
Comment (alhurwitz) [inline README.md:1]: Do we want to add instructions on how to get data with make? What about setting up AWS CLI?

**Review by alhurwitz — COMMENTED** [review]
Everything looks good to me! Let me know what you think about adding more instructions on readme!

File: README.md (hunk 1 above)
Comment (hilahersz) [reply README.md:14]: we'll have only after it's published

File: README.md
Comment (hilahersz) [reply README.md:1]: yes!

**Pull request merged**